.github/ISSUE_TEMPLATE/add-instance.md

```diff
@@ -6,9 +6,20 @@ labels: instance add, instance
 assignees: ''
 
 ---
+<!-- please answer the following questions (replace [ ] by [x] to say yes) -->
+* [ ] It is my instance.
+* [ ] I agree searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
+* [ ] My instance runs unmodified searx source code.
 
 <!--
-Replace <searx instance url> in the title by the URL.
+If you don't agree searx.space to check your instance, currently it is not possible to add your instance to the list.
+
+searx.space source code: https://github.com/dalf/searx-stats2
+
+If you have modified searx, please provide an URL to the source code, either here or in the theme of your instance.
```

Comment:
According to the AGPLv3 those who modify the source code must have a public URL in their site to their full code. Thus I'd suggest to change this text to:
```
If you have modified searx, please provide an URL to the source code and make sure that the same URL is publicly accessible in your instance's site.
```

Reply:
I removed the question "My instance runs unmodified searx source code.". It seems an yes answer is mandatory to get the instance accepted.

About LGPL: I think it should be solve in the searx project, but a reminder here won't hurt, so I added the text you have suggested.